Mathlib/Algebra/Order/Kleene.lean

```diff
@@ -324,7 +324,7 @@ end Prod
 namespace Pi
 
 instance instIdemSemiring [∀ i, IdemSemiring (π i)] : IdemSemiring (∀ i, π i) :=
-  { Pi.semiring, Pi.semilatticeSup, Pi.orderBot with
+  { Pi.semiring, Pi.instSemilatticeSup, Pi.instOrderBot with
```

Comment:
Now that instances are not protected by default (and have names that let them stay unprotected), I think we should move towards unqualifying the names
```suggestion
  { semiring, instSemilatticeSup, instOrderBot with
```

Reply:
I'd rather leave that to a follow-up; it's much easier to do a replacement of  `Foo.instBar` with `instBar` via a regex later.

It's also not fully clear to me whether unprotected instances is the right choice.